CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+## 1.2.0
+
+## Features and improvements
+* Add *-sort-key layout property for circle, fill, line ([#8467](https://github.com/mapbox/mapbox-gl-js/pull/8467))
+* Add map-specific access tokens ([#8364](https://github.com/mapbox/mapbox-gl-js/pull/8364))
```

Comment:
What does this do? If it's not that important, it should be lower in the changelog; if it is that important, its importance and impact should be clearer in the changelog entry

Reply:
It enables users to add maps with access tokens specific to the map, but I'm struggling to make that more clear beyond switching the words around 🤔as for importance, there aren't many other big feature enhancements in this release overall, hence the prioritization